src/__swaps__/screens/Swap/components/SearchInput.tsx

```diff
@@ -80,13 +81,21 @@ export const SearchInput = ({
   const onInputSearchQueryChange = useDebouncedCallback(
     (text: string) => {
       userAssetsStore.getState().setSearchQuery(text);
+      analyticsV2.track(analyticsV2.event.swapsSearchedForToken, {
```

Comment:
This is going to be very noisy, seems a little excessive. Would suggest we capture the search query (if there is one) when an asset is selected or on input blur. Same for output search tracking below.

Reply:
Yeah, I agree. Even if it's just adding to the batch (and not making a network request) it's not something we want to track on every key press for performance reasons and data pollution on the analytics side.